cmake/format.cmake

```
@@ -21,7 +21,7 @@
 # format`. To only check whether the codebase is properly formatted, run `make
 # check-format` instead.
 
-set(FORMATTING_SOURCES apps/**/*.cpp  apps/**/*.hpp tests/*.cpp include/**/*.hpp)
+set(FORMATTING_SOURCES examples/**/*.cpp  examples/**/*.hpp tests/*.cpp include/**/*.hpp)
```

Hmm, seems to me as if we need to make the path more precise here for the formatting to work.

```suggestion
set(FORMATTING_SOURCES examples/wordCount/**/*.cpp tests/*.cpp include/**/*.hpp)
```

---

I found a somewhat more generic solution for this in the comments beneath https://stackoverflow.com/a/36046965, commit abd7a17a3aba48a77fc4d829d107e299eaed5180 should fix the issue.

(also, I think the way `FORMATTING_SOURCES` was set before did not actually include all source files, `**` seems to not be expanded as one would expect)